app/services/idv/analytics_events_enhancer.rb

```diff
@@ -183,8 +183,20 @@ def profile_history
     end
 
     def proofing_components
-      return if !user&.respond_to?(:proofing_component) || !user.proofing_component
-      ProofingComponentsLogging.new(user.proofing_component)
```

Comment:
Do we still need `ProofingComponentsLogging`? I was looking for the red in this diff and didn't see it.

Reply:
No we don't, I can remove that

Reply:
Removed in 131de1f, with a few extra commits for analytics doc cleanup